src/main/java/cz/cvut/kbss/study/model/User.java

```diff
@@ -61,9 +62,22 @@ public class User implements HasDerivableUri, Serializable {
     @OWLObjectProperty(iri = Vocabulary.s_p_is_member_of, fetch = FetchType.EAGER)
     private Institution institution;
 
+    @OWLDataProperty(iri = Vocabulary.s_p_role_group)
```

Comment:
Most-likely we will need OWLObjectProperty here [1].

Reply:
done